Bottom: 58e240d556f93a7705ae97d17d26d011f6281026
Top:    341fa1c4551cbd626ae75a594d096d70d8cbbd8e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-15 01:46:48 +0000

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 96e2473..8f91ba6 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   ghid_set_lock_effects (Output.fgGC, (AnyObjectType *)element);
